packages/okta-react/test/e2e/harness/src/CustomLogin.js

```diff
@@ -14,8 +14,10 @@ import { Component } from 'react';
 import { withAuth } from '@okta/okta-react';
 
 export default withAuth(class CustomLogin extends Component {
-  render() {
+  componentDidMount() {
```

Comment:
Not sure I understand why these changes are needed (wasn't expecting tests changes in this PR), how passing from yarn workspace to lerna changes test behavior?

Reply:
Performing the redirect in render() is a problem if the component renders more than once.  I experienced this during debugging. I believe it will usually render twice, which causes the first redirect request to cancel.

Reply:
I believe this used to work in auth-js < 2.6 because `getWithRedirect` was synchronous. Now there it is async and there is time for the 2nd request to fire.